src/ts/components/sign-up/onboarding/screens/index.tsx

```diff
@@ -1,450 +1,8 @@
-    <ScreenTemplate
-      textAlign="center"
-      before={<AppIcon />}
-      title={`You’re all set! ${appName} has been connected to your Data Vault`}
-      body={[`Everything you do in ${appName} will be private, secure, and only accessible with your Secret Key.`]}
-      action={{
-        label: 'Done',
-        onClick: props.next,
-      }}
-    />
-  );
-};
-
-interface SignInProps {
-  next: () => void;
-  back: () => void;
-}
-
-const SignIn: React.FC<SignInProps> = props => {
-  const [isLoading, setLoading] = useState(false);
-  const [seed, setSeed] = useState('');
-  const [seedError, setSeedError] = useState<null | string>(null);
-  const dispatch = useDispatch();
-  const appName = useSelector((state: IAppState) => selectAppName(state));
-
-  return (
-    <ScreenTemplate
-      textAlign="center"
-      before={<AppIcon />}
-      title={`Sign into ${appName}`}
-      body={[
-        'Enter your Data Vault’s Secret Key to continue',
-        <Box textAlign="left">
-          {/*Validate: track SIGN_IN_INCORRECT*/}
-          <Input
-            autoFocus
-            minHeight="80px"
-            placeholder="12-word Secret Key"
-            as="textarea"
-            value={seed}
-            onChange={(evt: React.FormEvent<HTMLInputElement>) => {
-              setSeedError(null);
-              setSeed(evt.currentTarget.value);
-            }}
-          />
-          {seedError && (
-            <Text textAlign="left" textStyle="caption" color="feedback.error">
-              {seedError}
-            </Text>
-          )}
-        </Box>,
-      ]}
-      action={[
-        {
-          label: 'Create a Data Vault',
-          variant: 'text',
-          onClick: () => {
-            doTrack(SIGN_IN_CREATE);
-            dispatch(doChangeScreen(Screen.INTRO));
-          },
-        },
-        {
-          label: 'Continue',
-          onClick: async () => {
-            setLoading(true);
-            try {
-              if (seed.trim().split(' ').length <= 1) {
-                dispatch(doSetMagicRecoveryCode(seed.trim()));
-                dispatch(doChangeScreen(Screen.RECOVERY_CODE));
-                return;
-              }
-              await doStoreSeed(seed, DEFAULT_PASSWORD)(dispatch, () => ({}), {});
-              doTrack(SIGN_IN_CORRECT);
-              props.next();
-            } catch (error) {
-              setSeedError("The seed phrase you've entered is invalid.");
-              doTrack(SIGN_IN_INCORRECT);
-            }
-            setLoading(false);
-          },
-        },
-      ]}
-      isLoading={isLoading}
-      footer={
-        <>
-          <Flex>
-            <Link onClick={() => doTrack(SIGN_IN_FORGOT)}>Forgot Secret Key?</Link>
-          </Flex>
-          <Link>Help</Link>
-        </>
-      }
-    />
-  );
-};
-
-export { Intro, HowItWorks, Create, SecretKey, Connect, SaveKey, Final, SignIn };
+import { SignIn } from '@components/sign-up/onboarding/screens/sign-in';
```

Comment:
Nit: could use the `export { x } from ...` syntax and remove the export at the end